js/libs/ui-shared/src/controls/TextAreaControl.tsx

```diff
@@ -13,12 +13,14 @@ import { KeycloakTextArea } from "./keycloak-text-area/KeycloakTextArea";
 export type TextAreaControlProps<
   T extends FieldValues,
   P extends FieldPath<T> = FieldPath<T>,
-> = UseControllerProps<T, P> &
-  TextAreaProps & {
-    label: string;
-    labelIcon?: string;
-    isDisabled?: boolean;
-  };
+> = UseControllerProps<T, P> & {
+  label: string;
+  labelIcon?: string;
+  isDisabled?: boolean;
```

Comment:
What is the difference between `isDisabled` and `readOnly`?

Reply:
Good question @jonkoops As far I as I can tell there is only a css difference -> text colour is light grey when isDisabled and black when readOnly.